pynxtools/_build_wrapper.py

```diff
@@ -0,0 +1,66 @@
+"""
+Build wrapper for setuptools to create a nexus-version.txt file
+containing the nexus definitions verison.
+"""
+import os
+from subprocess import CalledProcessError, run
+from typing import Optional
+
+from setuptools import build_meta as _orig
+from setuptools.build_meta import *  # pylint: disable=wildcard-import,unused-wildcard-import
+
+
+def get_vcs_version(tag_match="*[0-9]*") -> Optional[str]:
+    """
+    The version of the Nexus standard and the NeXus Definition language
+    based on git tags and commits
```

Comment:
what about the git repo used?

Reply:
mmh... I did not think about this yet as I thought we have it fixed here anyways. But maybe it's a good idea to determine it from the remotes automatically, too (so we don't have to change anything if the repo changes).

Reply:
Let's include this in the TF discussion and then I can add it.

Reply:
Currently, we only get the vcs version here, but we add a repository with the current hash along the actual version. I also added `/@NeXus_repository` to NXroot (which I still need to add to NXroot) for this.